docs/reference/automatic-instrumentation.md

```diff
@@ -64,3 +73,63 @@ plugins {
 ```
 
 1. You can find the latest version [here](https://plugins.gradle.org/plugin/co.elastic.otel.android.instrumentation.okhttp).
+
+## OpenTelemetry Android Instrumentation Adapter
+
+**Status**: [experimental](component-stability.md#experimental)
+
```

Comment:
````suggestion
```{applies_to}
product: experimental
```
````

Reply:
Can we use the `{applies_to}` approach and still keep the link to the experimental status definition?

Reply:
I think we can! Let me edit